src/games/gamemodelitem.py

```diff
@@ -15,13 +16,34 @@ def __init__(self, game, me, preview_dler):
 
         self.game = game
         self.game.gameUpdated.connect(self._game_updated)
+        self.hosted_at = None  # put here as long it's not coming from server
         self._me = me
         self._me.relationsUpdated.connect(self._check_host_relation_changed)
         self._preview_dler = preview_dler
         self._preview_dl_request = PreviewDownloadRequest()
         self._preview_dl_request.done.connect(self._at_preview_downloaded)
 
+        # Update highlight
```

Comment:
The complete lack of comments explaining what this "highlight" logic is trying to achieve is unhelpful.